cli/test/test_parser.py

```diff
@@ -633,3 +633,37 @@ def test_complete_graph(self):
                 'name': 'end'}
         }
         self.assertDictEqual(wf.steps, steps_dict)
+
+
+class TestPopperConfigParser(unittest.TestCase):
```

Comment:
maybe move this to a new test_config.py?

Reply:
Done